.pre-commit-config.yaml

```diff
@@ -0,0 +1,53 @@
+---
+# Copyright (C) 2023 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+#
+# Use `pre-commit autoupdate` to update this file.
+repos:
+  - repo: "https://github.com/pre-commit/pre-commit-hooks"
+    rev: v4.4.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-ast
+      - id: check-docstring-first
+      - id: check-executables-have-shebangs
+      - id: check-merge-conflict
+      - id: check-shebang-scripts-are-executable
+      - id: check-symlinks
+      - id: check-toml
+      # - id: check-yaml
+      - id: destroyed-symlinks
+      - id: debug-statements
+      - id: end-of-file-fixer
+      - id: fix-byte-order-marker
+      - id: fix-encoding-pragma
+        args:
+          - "--remove"
```

Comment:
Can you add a reference/PEP that states that omitting the `coding` line is not best practice?

Reply:
There's a nice SO answer here: https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python-3 .

I believe that the whole repository is UTF-8, and all our tools are UTF-8 by default. In the case of vim, the "-*- coding: latin-1 -*-" comment itself has no effect as it's not a correct vim line.

Reply:
Ok. I have no strong opinions about this so I am fine with that.